samples/tutorial/Tutorial2.md

```diff
@@ -471,11 +542,11 @@ object RootWorkflow : StatefulWorkflow<Unit, State, Nothing, BackStackScreen<Any
 
       // When the state is Todo, defer to the TodoListWorkflow.
       is Todo -> {
-        val todoListScreens = context.renderChild(TodoListWorkflow, props = ListProps(state.username)) {
+        val todoListScreen = context.renderChild(TodoListWorkflow, props = ListProps(state.username)) {
```

Comment:
this becomes plural in the next tutorial. The type here is `___Screen` not `List<___Screen>`